packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -925,6 +925,29 @@ const String kIosArchs = 'IosArchs';
 /// Supported values are x86_64 and arm64.
 const String kDarwinArchs = 'DarwinArchs';
 
+/// The define to control what Android architectures are built for.
+///
+/// This is expected to be a space-delimited list of architectures.
+const String kAndroidArchs = 'AndroidArchs';
+
+/// If the current build is `flutter build aar`.
+///
+/// This is expected to be a boolean.
+///
+/// If not provided, defaults to false.
+const String kIsAddToApp = 'IsAddToApp';
+
+/// The define to control what min Android SDK version is built for.
+///
+/// This is expected to be int.
+///
+/// If not provided, defaults to `minSdkVersion` from gradle_utils.dart.
+///
+/// For more info, see:
+/// https://developer.android.com/ndk/guides/sdk-versions#minsdkversion
+/// https://developer.android.com/ndk/guides/other_build_systems#overview
+const String kMinSdkVersion = 'MinSdkVersion';
```

Comment:
Please include where this minsdkversion is derived from. The concept makes sense but information a future debugger is going to want/need is where this minSdkVersion comes from and why it is different from what they expect. 

I am guessing this is a cache for a value that is evaluated and passed build_info.